GNUmakefile

```
@@ -144,6 +144,12 @@ presubmit: ## Format go files and imports.
 gomods: ## Install gomods
 	go install github.com/jmank88/gomods@v0.1.4
 
+.PHONY: gomodrequiredupdater
+gomodrequiredupdater: ## Run gomod-required-updater
+	go install ./tools/gomod-required-updater/cmd/gomod-required-updater
+	gomods -w gomod-required-updater
```

Can we find a better name? This is really about "local replaces". Maybe `gomodslocalupdate`?
```suggestion
gomodslocalupdate: ## Run gomod-required-updater
	go install ./tools/gomod-local-updater/cmd/gomod-local-updater
	gomods -w gomod-local-updater
```

---

I like this better. Updated in f17e56d19218e4be2a71599b2587ab3e5bece19e. Should I also update the tool name?

From: `./tools/gomod-required-updater`
To: `./tools/gomod-local-update`

---

Updated in 6c6c8be76631828630d224992d09a6431a0880b6 and 9ec395a3bfc0692b29febeb79daf8e735c9aaa1d.